pkgs/development/interpreters/php/default.nix

```diff
@@ -19,7 +19,8 @@ let
 
       enableParallelBuilding = true;
 
-      buildInputs = [ flex bison pkgconfig systemd ];
+      buildInputs = [ flex bison pkgconfig ]
+      ++ lib.optional stdenv.isLinux [systemd ];
 
```

Comment:
`--with-fpm-systemd` apparently, which tells it to manage its fastcgi processes via systemd